imports/plugins/core/checkout/client/components/cartItems.js

```diff
@@ -5,7 +5,6 @@ import { Components, registerComponent } from "@reactioncommerce/reaction-compon
 class CartItems extends Component {
   static propTypes = {
     handleImage: PropTypes.func,
-    handleLowInventory: PropTypes.func,
```

Comment:
Good catch. I removed it intending to add it back after I figured out how to do it. I'll add it back now using the `isLowQuantity` flag.

Reply:
Actually @spencern, I remember now. I don't think this was working correctly because it was using the `item.variants.inventoryQuantity` to determine whether to show, but that value becomes static and isn't updated as other people continue to purchase that variant. So the fix would be to look up from the Products collection I think, which would require making sure all the cart variant products are published if they aren't. Or we could make a method that returns the flag and call that. Is that worth doing given that we are deprecating this UI?

Reply:
If it's not breaking existing functionality, I don't know that we need to put much time into adding this to our Meteor storefront.